sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -129,37 +129,37 @@ The Text Analytics client library provides a [TextAnalyticsClient][text_analytic
 [TextAnalyticsAsyncClient][text_analytics_async_client] to do analysis on batches of documents. It provides both synchronous and
 asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction.
 
-### Text input
-A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models
-in the Text Analytics service. Operations on Text Analytics client may take a single text input or a collection
-of inputs to be analyzed as a batch. 
+### input
```

Comment:
Sorry for changing what I had correctted earlier. I think this should be

### Document
A **document** is a text input to be analyzed by the predictive models

Reply:
I agree with Isa on the text "A document is a text input to be analyzed by the predictive models"

I do consider we should leave the tittle of the section to be "Input". That way this section has client, input, return value, etc

Reply:
No worries. Thank you for checking my grammars lol. See above comment.